tests/sentry/ingest/ingest_consumer/test_ingest_consumer_processing.py

```diff
@@ -161,6 +161,86 @@ def test_transactions_spawn_save_event_transaction(
     )
 
 
+@django_db_all
```

Comment:
you should probably write unit tests for `event_tracker.py`. i don't think you need tests for this in every ingest consumer, especially if this is supposed to be extended and used in a bunch of places later.

Reply:
if it's just unit tests for `event_tracker.py` what should the tests be testing? Wouldn't that just be is logging called or not?